libparsec/crates/testbed/src/templates/workspace_history.rs

```diff
@@ -48,6 +48,10 @@ use crate::TestbedTemplate;
 /// - `foo/spam`:
 ///   - 2001-01-08T03:00:00: v1 from Mallory, content: []
 pub(crate) fn generate() -> Arc<TestbedTemplate> {
+    // If you change something here:
+    // - Update this function's docstring
+    // - Update `server/tests/common/client.py::WorkspaceHistoryOrgRpcClients`s docstring
```

Comment:
See my other comment in `server/tests/common/client.py`

```suggestion
    // If you change something here, please update the docstring
    //
    // This template is exposed in:
    // `server/tests/common/client.py::WorkspaceHistoryOrgRpcClients`
```